src/test/HostActivationTests/DotNetBuilder.cs

```diff
@@ -96,18 +96,20 @@ public DotNetBuilder AddMicrosoftNETCoreAppFrameworkMockCoreClr(string version)
 
             string netCoreAppPathDepsJson = Path.Combine(netCoreAppPath, "Microsoft.NETCore.App.deps.json");
 
+            string currentRid = PlatformAbstractions.RuntimeEnvironment.GetRuntimeIdentifier();
```

Comment:
We typically use `TestProjectFixture.CurrentRid` as it allows tests to be targeted to different RIDs than what is the exact current machine.

Reply:
`DotNetBuilder` doesn't have a `TestProjectFixture`.  I used `_repoDirectories.TargetRID` which is what initializes `TestProjectFixture.CurrectRid`